sitewit/models.py

```diff
@@ -8,9 +12,17 @@ def __init__(self, name, email, token):
         self.token = token
 
 
-class Account(object):
+class SiteWitServiceModel(object):
```

Comment:
It wasn't a name change, diff is a bit weird here.  I've just added a base model.  (not sure if I named it properly thou)